operator_ui/package.json

```diff
@@ -45,22 +45,21 @@
     "path-to-regexp": "^3.0.0",
     "promise.prototype.finally": "^3.1.0",
     "prop-types": "^15.6.2",
-    "react": "^16.7.0",
+    "react": "^16.8.4",
```

Comment:
Any reason we can't upgrade to .12 while at it? https://github.com/facebook/react/blob/master/CHANGELOG.md#16120-november-14-2019

Reply:
I don't see one especially since that's what we're using according to the yarn.lock file anyway